seqr/views/utils/orm_to_json_utils.py

```diff
@@ -181,6 +181,13 @@ def _process_result(result, family):
             result['individualGuids'] = [i.guid for i in family.individual_set.all()]
         if not result['displayName']:
             result['displayName'] = result['familyId']
+        if result['assignedAnalyst']:
+            result['assignedAnalyst'] = {
+                'fullName': result['assignedAnalyst'].get_full_name(),
+                'email': result['assignedAnalyst'].email,
+            }
+        else:
+            result['assignedAnalyst'] = ''
```

Comment:
Its probably better to use `None` here, because thats the "falsey" version of objects and its better to always have the value of a field have the same type, and while the empty string is always falsey it is technically a different type